src/ai/backend/manager/models/scaling_group.py

```diff
@@ -277,6 +277,11 @@ class ScalingGroupRow(Base):
         "ScalingGroupForKeypairsRow",
         back_populates="sgroup_row",
     )
+    resource_preset_rows = relationship(
+        "ResourcePresetRow",
+        back_populates="scaling_group_row",
+        primaryjoin="ScalingGroupRow.name == foreign(ResourcePresetRow.scaling_group_name)",
+    )
```

Comment:
I'm not sure if you need to join based on the name, as it's not an ID value that needs to be joined.

Reply:
Sadly, `scaling_groups` table does not have id column and it uses `name` column as a primary key